cli/flox/src/commands/activate.rs

```diff
@@ -212,6 +213,12 @@ impl Activate {
                 "Environment is already active: environment={}. Not adding to active environments",
                 now_active.bare_description()?
             );
+            if interactive {
+                return Err(anyhow!(
+                    "Environment {} is already active",
+                    now_active.message_description()?
```

Comment:
@zmitchell currently we're passing `bare_description()` for `FLOX_ENV_DESCRIPTION`, do we want to preserve that? I didn't cause this is a message not a prompt